hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/MiniDFSCluster.java

```diff
@@ -2815,23 +2824,11 @@ public Boolean get() {
    */
   public void waitActive() throws IOException {
     for (int index = 0; index < namenodes.size(); index++) {
-      int failedCount = 0;
       while (true) {
```

Comment:
After this change, while loop is not required because `waitActive(index)` would take care of internal retries if required.

Reply:
I pulled this loop inside the waitActive(index) itself, this is the loop which was handling the failedCounts